### PR TITLE
Add a credits page listing the licenses for Tofino, Electron, ffmpeg and all the libraries that Chromium depends on.

### LINKED ISSUE #293: Have an about:license page

### PR DESCRIPTION
**Pull request by Mossop**
Fixes #293

### CODE REVIEW

**Comment by Mossop** [comment]
See the credits page this adds here: https://rawgit.com/Mossop/tofino/5a07c7a90ac9e60e65d163f45ce529bb4f56a9ae/app/ui/about/credits.html

**Comment by victorporof** [comment]
I believe this needs a rebase now that we're serving about pages over HTTP.

**Comment by Mossop** [comment]
Legal have approved the credits page so we can land this if the code looks good.

File: app/main/about-pages.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * app's 'ready' event.
+ *
+ * @TODO considerations:
+ * @TODO Use `protocol.registerHttpProtocol` so we can redirect invalid requests
+ * to default search?
+ * @TODO Proxy valid `chrome://*` URIs to `tofino://*`?
+ *   https://github.com/electron/electron/issues/2618
+ */
+export default function() {
+  protocol.registerFileProtocol(ABOUT_PROTOCOL_PREFIX, (request, callback) => {
```
Comment (victorporof) [inline app/main/about-pages.js]: Should either register a http protocol now that that works, or make use of the fact that we're serving static content already via express, so no need to register a file protocol, right?

Comment (Mossop) [inline app/main/about-pages.js]: Oh I think this ended up in here because of a weird rebase experience. I'll get rid of it.

Comment (victorporof) [inline app/main/about-pages.js]: Feel free to land afterwards.

**Pull request merged**